package/gluon-check-connection/Makefile

```diff
@@ -0,0 +1,13 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-check-connection
```

Comment:
A better name could be `gluon-online-status`.

Reply:
check-connection is more generic, It could also check other connections thatn just online later

Reply:
I'd rather see it as a service that provides the online status to other packages. The scope should probably end there.